### ISSUE TITLE
Convert the remaining fixtures

### ISSUE THREAD

**Issue by gravitystorm** [gravitystorm/openstreetmap-website#4]
- [x] acls
- [x] changeset_comments
- [x] changesets_subscribers
- [ ] changesets
- [x] changeset_tags
- [x] client_applications
- [x] countries
- [ ] current_nodes
- [x] current_node_tags
- [ ] current_relation_members
- [ ] current_relations
- [ ] current_relation_tags
- [ ] current_way_nodes
- [ ] current_ways
- [ ] current_way_tags
- [x] friends
- [x] gps_points
- [x] gpx_files
- [x] gpx_file_tags
- [x] languages
- [x] messages
- [ ] nodes
- [x] node_tags
- [x] note_comments
- [x] notes
- [x] redactions
- [ ] relation_members
- [ ] relations
- [x] relation_tags
- [x] user_blocks
- [x] user_preferences
- [ ] user_roles
- [ ] users
- [x] user_tokens
- [ ] way_nodes
- [ ] ways
- [x] way_tags

`redactions` blocked by `nodes` blocked by `node_tags`
`changesets` blocked by `changeset_tags`

**Comment by gravitystorm** [comment]
Instructions are at https://gist.github.com/gravitystorm/80dc264af80c6bcac7e0cb110ae32256

**Comment by gravitystorm** [comment]
I've started working on the users fixtures. Given the size of this task I'm doing it one bit at a time, so the users factory is there, but many tests still use the fixtures.

When users is done, most parts of the site will be free from fixtures. The nodes/ways/relations stuff is rarely developed on so it can come afterwards.

**Comment by gravitystorm** [comment]
Jeez, this users factory thing is going to take a while. Before I started there were 828 calls to users() fixtures. 4 pull requests so far, but still 687 remaining.

`grep -rn 'users(' test/`

**Comment by gravitystorm** [comment]
Another PR, 620 calls remaining.

**Comment by gravitystorm** [comment]
Another PR today, 343 remaining.

**Comment by gravitystorm** [comment]
Two more PRs, one of which took a bit of headscratching to figure out. 272 remaining.

**Comment by gravitystorm** [comment]
73 users fixtures remaining. I've had to branch out here and work on other factories, since many of the remaining uses are tied to e.g. the owner of the (changeset for the) node/way/relation in a particular test, and I can't replace the user without a factory for the node/way/relation. So the pace of users fixtures have slowed, but I'm chipping away at the other fixtures too.

**Comment by gravitystorm** [comment]
OK, all the factories are now available and working correctly. Still 70 users fixtures remaining, but let's track things more widely. We're nearly in the endgame here:

```
nodes:      91
relations:  103
ways:       165
users:      70
changesets: 124

tests requiring fixtures:      178
```

**Comment by gravitystorm** [comment]
```
nodes:      71
relations:  103
ways:       77
users:      51
changesets: 113

total: 415
```

**Comment by gravitystorm** [comment]
```
nodes:      34
relations:  84
ways:       45
users:      42
changesets: 96

total: 301
```

**Comment by gravitystorm** [comment]
```
nodes:      19
relations:  39
ways:       39
users:      30
changesets: 88

total: 215
```

**Comment by gravitystorm** [comment]
```
nodes:      19
relations:  39
ways:       39
users:      2
changesets: 62

total: 161
```

**Comment by gravitystorm** [comment]
[Done!](https://github.com/openstreetmap/openstreetmap-website/pull/1556) :cake: :tada: :balloon: :white_check_mark:
